cmd/litcli/ln.go

```diff
@@ -182,6 +182,15 @@ var (
 			"set if there are multiple channels with the same " +
 			"asset ID present",
 	}
+
+	allowUnEconomicalFlag = cli.BoolFlag{
```

Comment:
How about `allowOverpay` or `allowAssetOverpay` or `allowTotalValueOverpay`

I think `allowTotalValueOverpay` is most clear.

Reply:
Couldn't see this change in the latest commits. Are they incoming?